SOPT-iOS/Projects/Features/HomeFeature/Sources/HomeScene/VC/HomeForMemberVC.swift

```diff
@@ -97,10 +97,22 @@ extension HomeForMemberVC {
         output.needToReload
             .withUnretained(self)
             .sink { owner, _ in
-                owner.collectionView.reloadData()
+                owner.updateUI()
             }.store(in: cancelBag)
     }
     
+    private func updateUI() {
+        updateCollectionViewLayout()
+        let sectionCount = HomeForMemberSectionLayoutKind.allCases.count
+        let indexSet = IndexSet(integersIn: 0..<sectionCount)
+        self.collectionView.reloadSections(indexSet)
+    }
```

Comment:
아래와 같이 reloadSections를 특정 섹션만 업데이트 하도록 하는 것은 어떨까요?

```swift
// ViewModel
    public struct Output {
        let needToReload = PassthroughSubject<IndexSet, Never>()
    }
```


```suggestion
    private func updateUI(_ sections: IndexSet) {
        updateCollectionViewLayout()
        self.collectionView.reloadSections(sections)
    }
```

위와 같이 구현 시 불필요한 섹션에 대한 렌더링을 줄일 수 있을 것 같아 제안드려요
이 코드리뷰가 뒷북이었거나 추후 구현 예정이었다면 넘겨주셔도 됩니다.

Reply:
오...! 생각해보니까 호출할 때 인덱스를 넘겨주면 되겠네요 ㅎ.ㅎ 감사합니닷

Reply:
@yungu0010 이 부분 API 모두 연동하고 추후에 구현하겠습니당